zerver/views/user_topics.py

```diff
@@ -66,10 +67,11 @@ def update_muted_topic(
     user_profile: UserProfile,
     stream_id: Optional[int] = REQ(json_validator=check_int, default=None),
     stream: Optional[str] = REQ(default=None),
-    topic: str = REQ(),
+    topic: str = REQ(converter=lambda var_name, data: data.strip()),
```

Comment:
How did you pick this `converter`? We don't use those often, and I'm reluctant to add a new one without tests, since I have a vague recollection of them having slightly different JSON decoding behavior?

Reply:
We use this converter in send-message codepath (see REQ_topic). Since we no longer plan to truncate & client should take care of such things; this is not needed and we can simply validate the topic length.